web-app/js/portal/details/LayerControlPanel.js

```diff
@@ -19,37 +19,50 @@ Portal.details.LayerControlPanel = Ext.extend(Ext.Container, {
             this.items.push(layerSelector, { xtype: 'spacer', height: 10 });
         }
 
-        this.items.push(this._newOpacitySlider());
+        this.opacitySlider = new Portal.common.LayerOpacitySliderFixed({
+            layer: this.layer,
```

Comment:
We specifically moved the creation of the controls into their own methods here to keep the constructor tidy. I suggest we put this control creation back into it's own method.

When we configure it we can pass a `listeners` config option. Something like this:
``` javascript
listeners: {
    'changecomplete': this.doTracking,
    scope: this
}
```

So we wouldn't need to keep a reference to the slider (i.e. `this.opacitySlider`) at all.